src/calendar/styles.scss

```diff
@@ -56,11 +56,11 @@
 
   &-grid {
     inline-size: 100%;
+    table-layout: fixed;
```

Comment:
This rules gives all cells in a table the same width. This way, the same rule can be used for any number of row cells instead of specifying a specific fraction of the total (e.g, 1/7 or 1/3) at the cell level.

https://developer.mozilla.org/en-US/docs/Web/CSS/table-layout

https://caniuse.com/?search=table-layout

Reply:
Sounds good. Let's double-check that there's no regressions with a full screenshot test

Reply:
Ran the tests. There are some sub-pixel diffs in Firefox and Safari but they are stable across runs, not flaky. So they will require manual approval but only once.

![Screenshot 2024-02-14 at 17 11 49](https://github.com/cloudscape-design/components/assets/1257272/2a5e4675-194f-43db-8a53-c4775ddfab05)